osc-rest-server/src/main/java/org/osc/core/broker/rest/server/api/VirtualizationConnectorApis.java

```diff
@@ -492,5 +510,99 @@ public Response updateVirtualSecurityPolicyBindings(@Context HttpHeaders headers
         }
         return this.apiUtil.getResponseForBaseRequest(this.bindSecurityGroupService, bindRequest);
     }
+    
+    // Service Function Chain APIs
+    @ApiOperation(value = "Creates a Service Function Chain",
+            notes = "Creates a Service Fucntion Chain owned by Virtualization Connector provided and kicks off a " + "sync job",
+            response = BaseJobResponse.class)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "Successful operation"),
+            @ApiResponse(code = 400, message = "In case of any error", response = ErrorCodeDto.class) })
+    @Path("/{vcId}/serviceFunctionChain/{name}")
+    @POST
+    public Response createServiceFunctionChain(@Context HttpHeaders headers,
```

Comment:
dont take the name as a parameter. The Posted object(ServiceFunctionChainDto) should have all the details